.swagger-codegen-ignore

```diff
@@ -5,4 +5,17 @@ src/.php_cs
 src/git_push.sh
 src/test/**
 src/HeaderSelector.php
-src/ObjectSerializer.php
\ No newline at end of file
+src/ObjectSerializer.php
+
+# replaced by array and mixed types
+src/docs/Model/RawDeviceAttribute.md
+src/docs/Model/RawDeviceAttributeError.md
+src/docs/Model/RawDeviceAttributes.md
+src/docs/Model/Tag.md
+src/docs/Model/WebhookRawDeviceAttributes.md
+
+src/Model/RawDeviceAttribute.php
+src/Model/RawDeviceAttributeError.php
+src/Model/RawDeviceAttributes.php
+src/Model/Tag.php
+src/Model/WebhookRawDeviceAttributes.php
```

Comment:
Are we going to update these models and docs manually from now on?

Reply:
I just removed them by replacing with simple `array`. It was the most easy and correct in terms of PHP way.